source/fuzz/fuzzer_context.h

```diff
@@ -227,6 +228,10 @@ class FuzzerContext {
     return go_deeper_in_constant_obfuscation_(depth, random_generator_);
   }
 
+  // Requires |min_max.first| <= |min_max.second|, and returns a value in the
+  // range [ |min_max.first|, |min_max.second| ]
+  uint32_t ChooseBetweenMinAndMax(const std::pair<uint32_t, uint32_t>& min_max);
+
```

Comment:
We try to avoid adding generic functions, instead preferring to add many named functions specific to what is being requested.  The idea is that this might allow for finer-grained control in the future when we want to bias how values are chosen for specific purposes.

It thus might be better to leave this as private, and introduce a public method named according to its specific use case even if that public method just calls this private method.

Reply:
Fixed in 167cc6e.